spring-kafka-test/src/main/java/org/springframework/kafka/test/context/EmbeddedKafkaContextCustomizer.java

```diff
@@ -41,8 +41,9 @@
  * @author Elliot Metsger
  * @author Zach Olauson
  * @author Oleg Artyomov
+ * @author Sergio Lourenco
  *
- * @since 1.3
+ * @since 2.2.4
```

Comment:
You don't change this one at all. 
And you don't introduce any public API in this class, so only your name and Copyright.